File: 0338-counting-bits/0338-counting-bits.cpp
class Solution {
public:
    int countbit(int n){
        int count = 0;
        while(n){
            if(n&1)
                count++;
            n>>=1;
        }
        return count;
    }
    vector<int> countBits(int n) {
        vector<int>ans;
        for(int i = 0; i<=n; i++){
            ans.push_back(countbit(i));
        }
        return ans;
    }
};

File: 0338-counting-bits/0338-counting-bits.js
/**
 * @param {number} n
 * @return {number[]}
 */
var countbit = function(n){
    let count = 0;
        while(n){
            if(n&1)
                count++;
            n>>=1;
        }
        return count;
}
var countBits = function(n) {
    let ans = [];
        for(let i = 0; i<=n; i++){
            ans.push(countbit(i));
        }
        return ans;
};